h/views.py

```diff
@@ -1,16 +1,18 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 import logging
-import re
 
 from pyramid import httpexceptions
 from pyramid.events import ContextFound
 from pyramid.view import forbidden_view_config, notfound_view_config
 from pyramid.view import view_config
+import pyramid.response
 
 from . import session
 from .models import Annotation
 from .resources import Application, Stream
+import h.api_client
```

Comment:
I don't agree that it's a bad idea, I'm afraid, which I why I kept asking for it! It doesn't break `mock.patch`, it just means that you have to patch the object where it's used rather than where it is defined, which I've never really seen as a problem. If anything, it makes test isolation slightly stronger.

It also makes it very clear which modules are local imports, and makes it substantially easier to move packages and modules around if you need to.

Lastly, if we do decide we want to adopt a different style, then we should do that as group. For the time being, your PR should be consistent with the status quo, which in this file is explicit relative imports for local modules.